docs/supporting-internet-explorer-8.md

```diff
@@ -0,0 +1,108 @@
+# Supporting Internet Explorer 8
+
+If you are including GOV.UK Frontend as part of your application's stylesheets
+then you will need to do some additional work to support Internet Explorer 8.
+
+1. [Generate an IE8 specific stylesheet](#bundling-an-ie8-specific-stylesheet)
+2. [Transform the IE8 stylesheet using oldie](#transforming-the-ie8-stylesheet-using-oldie)
+3. [Include the IE8 stylesheet in your project](#including-the-ie8-stylesheet-in-your-project)
+
+You will then be able to [write CSS that targets IE8 in your own application
+styles](#writing-styles-that-target-ie8).
+
+If you are using the distributed versions of GOV.UK Frontend that already
+include an IE8 specific stylesheet then you can include that in your project.
```

Comment:
Change:

include an IE8 specific stylesheet

to:

include an IE8-specific stylesheet